samples/topology/src/main/java/quarks/samples/topology/CombiningStreamsProcessingResults.java

```diff
@@ -0,0 +1,158 @@
+/*
```

Comment:
Header for new files needs to be ASF licence header:

http://apache.org/legal/src-headers.html#headers